ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-stable.yaml

```diff
@@ -55,6 +55,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-byo-route53
+- as: aws-ipi-disconnected-cco-manual-sts-p2-f14
+  cron: 51 1 10,25 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @disconnected and @network-openshiftsdn and not
+        @fips'
+      TAG_VERSION: '@4.10'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service
```

Comment:
Why 4.10 amd64 stable has no aws-ipi-disconnected-cco-manual-sts-p3-f28-destructive like others?

Reply:
Maybe @liangxia can give a clear answer about this.

Reply:
Maybe we discussed the test strategy for stable testing, we are hoping most of functional testing should be well covered in nightly testing, stable testing would only include some basic testing. And here is the proposal about stable test strategy adjustment. https://docs.google.com/document/d/1YBYMFibMCI-C8jj6KZyyRiTTMb6js1YIwwjTMqDAxp0/edit#heading=h.qa2ebz7w17bq

Reply:
Ok. Thanks for explanations!